app/src/main/java/com/kickstarter/ui/fragments/BackingFragment.kt

```diff
@@ -65,8 +67,10 @@ class BackingFragment : BaseFragment<BackingFragmentViewModel.ViewModel>() {
                 .subscribe { setBackerImageView(it) }
 
         this.viewModel.outputs.backerName()
+                .observeOn(io())
```

Comment:
The changes for this subscription are related to this -> https://console.firebase.google.com/u/1/project/android-external-release/crashlytics/app/android:com.kickstarter.kickstarter/issues/310732673fa17a383580c28e897e2993?time=last-seven-days&versions=2.11.0%20(2011121700)&sessionEventKey=5FB659DF012C00017CA8F31D71F5CF52_1475252020298250304